.github/workflows/release.yaml

```diff
@@ -6,14 +6,16 @@ on:
       type:
         description: How the new release should be versioned
         required: true
+        type: choice
         options:
           - "major"
           - "minor"
           - "patch"
 
 jobs:
-  build:
+  publish:
     runs-on: ubuntu-latest
+    if: github.actor == 'piccirello'
```

Comment:
This workflow will be runnable by anyone with 'write' perms to the repo. To lock it down further, we can use an allowlist of users. I'm starting with myself and we can expand this. Ideally we could set a group, however I haven't found a way to do this.

Reply:
I think this can also be defeated by creating a branch and just changing this file (to run on every branch, e.g.). This seems like a huge gap but looking around StackOverflow and GitHub's docs, I couldn't find anyone talking about it.

Still, I think it's worth it to have the automation in place

Reply:
For posterity: this was solved with environments in https://github.com/DopplerHQ/cli/pull/337